### PR TITLE
Misc Improvements

### PR DESCRIPTION
**Pull request by Ashod**
Commit logs explain what these do. There are several cleanups, some perf improvements, some valgrind check fixes, and cleaning up RequestVettingStation instances (and capping them).

- wsd: use 'none' mount(2) fstype
- wsd: cosmetics
- wsd: avoid calling close(2) on invalid FD
- wsd: avoid unnecessary argument copying
- wsd: else-if mutually-exclusive condition
- wsd: test: protect against uninitialized unit-test
- wsd: avoid unnecessary TileDesc allocation
- wsd: include cleanup in COOLWSD.cpp
- wsd: reduce excessive 'using' keyword in COOLWSD.cpp
- wsd: remove const to move URI
- wsd: missing new-line in SocketPoll::dumpState
- wsd: cap in-flight RequestVettingStations
- wsd: StopWatch improvements
- wsd: track the age of RequestVettingStation instances
- wsd: clean up lingering RequestVettingStations
- wsd: trigger DocBroker cleaning up after exiting the poll

### CODE REVIEW

File: common/CoolMount.cpp [hunk 1]
```diff
@@ -310,6 +310,7 @@ int domount(int argc, const char* const* argv)
         {
             // Now we need to set read-only and other flags with a remount.
             unsigned long mountflags = (MS_BIND | MS_REMOUNT | MS_NODEV | MS_NOSUID | MS_RDONLY);
+            const char* fstype = "none";
```
Comment (caolanm) [inline common/CoolMount.cpp:313]: The ... "filesystemtype arguments are ignored", hopefully there isn't some annoying apparmor rule that will kick in with an unknown fstype.

File: wsd/ClientRequestDispatcher.hpp [hunk 2]
```diff
@@ -131,9 +134,19 @@ class ClientRequestDispatcher final : public SimpleSocketHandler
     /// WS is created and as long as it is connected.
     std::shared_ptr<RequestVettingStation> _rvs;
 
+    /// The minmum number of RVS instances in flight to trigger cleanup.
```
Comment (caolanm) [inline wsd/ClientRequestDispatcher.hpp:137]: typo, trivial

**Review by caolanm — APPROVED** [review]

File: common/CoolMount.cpp (hunk 1 above)
Comment (Ashod) [reply common/CoolMount.cpp:313]: Of course. This is to please the pedantic valgrind's check, which is bogus.

File: wsd/ClientRequestDispatcher.hpp (hunk 2 above)
Comment (Ashod) [reply wsd/ClientRequestDispatcher.hpp:137]: Good eye-sight!

**Comment by caolanm** [comment]
Ah, everything in here ended up in https://github.com/CollaboraOnline/online/pull/10294 and that's merged so don't need this then apparently.

**Pull request closed without merging**